image_generation/stable_diffusion_1_5/cpp/src/main.cpp

```diff
@@ -368,11 +368,11 @@ int32_t main(int32_t argc, char* argv[]) try {
 
     ov::Tensor text_embeddings = text_encoder(models, positive_prompt, negative_prompt);
 
-    std::shared_ptr<Scheduler> scheduler = std::make_shared<LMSDiscreteScheduler>();
-    scheduler->set_timesteps(num_inference_steps);
-    std::vector<std::int64_t> timesteps = scheduler->get_timesteps();
-
     for (uint32_t n = 0; n < num_images; n++) {
+        std::shared_ptr<Scheduler> scheduler = std::make_shared<LMSDiscreteScheduler>();
+        scheduler->set_timesteps(num_inference_steps);
+        std::vector<std::int64_t> timesteps = scheduler->get_timesteps();
```

Comment:
do we need the same for LCM ?
as I  see scheduler is created outside of the loop https://github.com/yatarkan/openvino.genai/blob/a21a9fcb7f1fbac5de239743567a677c63d4ab70/image_generation/lcm_dreamshaper_v7/cpp/src/main.cpp#L343-L348

Reply:
Just double checked LCM sample - it does work as is now with '--num' > 1 and generates normal images without moving scheduler inside the loop. I assume it is due to different implementations of `LMSDiscreteScheduler` and `LCMScheduler`

Reply:
I'm merging this since it resolves the issue. @yatarkan can you align the schedulers in a different PR